views/js/tools/bandwidth/tester.js

```diff
@@ -165,7 +188,8 @@ define([
      * @returns {Object}
      */
     var bandwidthTester = function bandwidthTester (config, diagnosticTool) {
-        var initConfig = getConfig(config || {}, _defaults);
+        var initConfig = getConfig(config, _defaults);
+        var level = Math.min(Math.max(parseInt(initConfig.level, 10), 1), _messages.length) - 1;
```

Comment:
This pattern is neither explicit (it took me while to read it) nor resilient as it creates an artificial link between the level value and the order in which the text are created in the array. Why not use an object container for the texts with the test id (`intensive_bandwidth`) as an getter? (or something like that, if possible)

Reply:
ok after discussion